ui/src/components/WMWorkerPoolEditor/index.jsx

```diff
@@ -82,6 +88,28 @@ export default class WMWorkerPoolEditor extends Component {
     saveRequest: func.isRequired,
     isNewWorkerPool: bool,
     deleteRequest: func,
+    /** Error to display when an action dialog is open. */
+    dialogError: oneOfType([string, object]),
+    /**
+     * Callback function fired when the DialogAction component throws an error.
+     * Required when viewing an existent address.
+     * */
+    onDialogActionError: func,
+    /**
+     * Callback function fired when the DialogAction component runs
+     * successfully. Required when viewing an existent address.
+     * */
+    onDialogActionComplete: func,
+    /**
+     * Callback function fired when the dialog should open.
+     * Required when viewing an existent address.
+     */
+    onDialogActionOpen: func,
+    /**
+     * Callback function fired when the dialog should close.
+     * Required when viewing an existent address.
+     */
+    onDialogActionClose: func,
```

Comment:
Many of these props have the comment `Required when viewing an existent address.`. This should be removed.